apps/playground/AGENTS.md

```
@@ -0,0 +1,140 @@
+pnpm exec biome check --write $(git diff --cached --name-only --diff-filter=ACM | grep -E '\.(ts|tsx|js|jsx)$')
+
+# 3. Stage the fixes
+git add -u
+
+# 4. Retry commit (NEVER use --no-verify)
+git commit -m "your message"
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)